.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -35,7 +37,19 @@ $tests = dotnet sln list |
     }
 
 foreach ($test in $tests) {
-    dotnet test -c Release --no-restore --no-build --nologo --logger "trx;LogFileName=test-results.trx" --verbosity $Verbosity $test 2>&1 >test.out
+    $dotnetTestSwitches = @(
+        '--configuration', 'Release'
+        '--no-restore',
+        '--no-build',
+        '--nologo',
+        '--logger', 'trx;LogFileName=test-results.trx'
+        '--verbosity', $Verbosity
+        [string]::IsNullOrEmpty($Filter) ? '' : "--filter"
+        [string]::IsNullOrEmpty($Filter) ? '' : $Filter
```

Comment:
`[string]::IsNullOrEmpty` is redundant, also the string quotes are inconsistent.
```suggestion
        $Filter ? "" : "--filter"
        $Filter ? "" : $Filter
```

Reply:
All of them should actually be single quote (since we don't need string interpolation).

Reply:
It's not really working as expected:

![image](https://user-images.githubusercontent.com/1976647/201495643-0da44f95-e166-4873-b993-72a99db5ae30.png)

I'm reverting to the original code

Reply:
Yeah, should be the other way around, since "truthy" implies that it's not null or empty.

Reply:
Eh.